docs/configuration/config-file-reference.md

```diff
@@ -613,6 +613,18 @@ lifecycler:
 # Period with which to update the per-user ingestion rates.
 # CLI flag: -ingester.rate-update-period
 [rate_update_period: <duration> | default = 15s]
+
+# Enable tracking of active series.
+# CLI flag: -ingester.active-series-enabled
+[active_series_enabled: <boolean> | default = false]
```

Comment:
Should we prefix all such config options with `active_series_metrics` to make it more clear?